src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,32 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import javax.swing.*;
+import java.time.Duration;
+
+public class BasePage {
+    public  WebDriver driver;
+    public WebDriverWait wait;
+    public String url;
+    Actions action;
+    public BasePage(WebDriver givenDriver){
+        driver = givenDriver;
+        wait = new WebDriverWait(driver, Duration.ofSeconds(15));
+        action= new Actions(driver);
+
+    }
+    public void doubleClickAxn(WebElement locator){
+        wait.until(ExpectedConditions.elementToBeClickable(locator));
+        action.doubleClick(locator).build().perform();
+}
+     public WebElement findElement(By locator){
+   return  wait.until(ExpectedConditions.visibilityOfElementLocated(locator));
+}
+}
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)